app/src/main/java/com/einmalfel/podlisten/MainActivity.java

```diff
@@ -225,6 +225,8 @@ void updateFAB(Pages fragment, float positionOffset) {
       case SUBSCRIPTIONS:
         lerpFAB(FabAction.ADD, FabAction.ADD, 0);
         break;
+      default:
+        break;
```

Comment:
We don't expect any other values here, so IMHO it's better to throw AssertionError or log an error instead of just "break;"